data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1021,20 +1021,114 @@ mission "Hai Reveal [B11] Moving Together"
+		"engine capacity" 139
+		# Expensive features inherited from Ladybug-era technology, with some fuel (and improved heat dissipation):
+		"heat dissipation" 0.7
+		"tactical scan power" 30
+		ramscoop .5
+		"fuel capacity" 600
+		weapon
+			"shield damage" 5000
+			"hull damage" 2500
+			# High hit force and wide radius for a spectacle if the leader dies.
+			# This should be adjusted downward if the ship is reused more widely.
+			"blast radius" 800
+			"hit force" 8000
 	outfits
-		"Large Heat Shunt" 2
+		"Tripulse Shredder" 2
+		"Ionic Blaster Prototype" 4
+		"Pulse Turret" 4
+		"Warder Anti-Missile" 2
+		
+		"Large Heat Shunt"
+		"Fuel Processor"
 		"Jump Drive"
-		"Systems Core (Large)"
 		"Quantum Keystone"
-		"Outfits Expansion" 3
-		"Boulder Reactor" 3
-		`"Bufaer" Atomic Steering`
-		Sunbeam 6
-		"Korath Warder"
-		"Small Heat Shunt"
+		"Systems Core (Large)"
+		"Systems Core (Medium)"
+		"Wanderer Ramscoop"
+		
+		"White Sun Reactor"
+		"Large Biochemical Cell"
+		
+		"Afterburner (Comet Class)"
+		"Steering (Planetary Class)"
 		"Thruster (Planetary Class)"
-		"Fuel Processor" 2
+		
+		"Pulse Rifle" 65
+		"Fragmentation Grenades" 65
+		"Thermal Repeater Rifle" 25
+	engine -43 131
+		zoom 0.85
+	engine 43 131
+		zoom 0.85
+	engine 0 126.5
+		zoom 0.5
+	"reverse engine" -61 -18.5
+		angle -35
+	"reverse engine" 62 -18.5
+		angle 35
+	"steering engine" -61 -18.5
+		zoom 0.6
+		angle 145
+		left
+	"steering engine" 62 -18.5
+		zoom 0.6
+		angle 215
+		right
+	"steering engine" -73 46
+		zoom 0.5
+		angle 80
+		left
+	"steering engine" 73 46
+		zoom 0.5
+		angle -80
+		right
+	"steering engine" -43 131
+		zoom 0.45
+		right
+	"steering engine" 43 131
+		zoom 0.45
+		left
+	gun -42.5 -53.5 "Tripulse Shredder"
+	gun 43.5 -54.5 "Tripulse Shredder"
+	gun -23 -102.5
+	gun 24.5 -102.5
+	gun 12.5 -128
+	gun -10 -128
+	turret 0.5 -25.5 "Warder Anti-Missile"
+	turret -47.5 11.5
+	turret 48.5 11.5
+	turret 62.5 76
+	turret -61.5 76.5
+	turret 0 107.5 "Warder Anti-Missile"
+	leak "leak" 35 45
+	leak "flame" 60 95
+	leak "big leak" 50 45
+	explode "tiny explosion" 60
+	explode "small explosion" 51
+	explode "medium explosion" 54
+	explode "large explosion" 53
+	explode "huge explosion" 22
+	"final explode" "final explosion large"
+	description "Before the Shield Beetle, the Hai used the Titan Beetle, a weaker but more capable ship of a similar design. The Emperor Beetle is a Titan coated with many layers of modern armor, making it a more powerful warship than anything else the Hai have built in modern times. Unfortunately, the advanced features are quite expensive, so these ships are used sparingly as flagships and trophy ships."
+	description "	As it is not meant to plunder but to lead in glorious battle, much of the cargo space was removed and converted to space for weapons and defensive improvements. This change came at the cost of increased mass and lowered agility, but that is a sacrifice the Unfettered are more than willing to make."
```

Comment:
```suggestion
	description "	As it is not meant to plunder but to lead in glorious battle, much of the cargo was removed and converted to space for weapons and defensive improvements. This change came at the cost of increased mass and lowered agility, but that is a sacrifice the Unfettered are more than willing to make."
```

Reply:
Gonna disagree with this one. Removing "cargo" is vastly different from removing "cargo space," and implied distinction between the two only comes afterwards in the sentence.

It is a bit repetitive, but this makes it unclear instead.

Reply:
Maybe swap it for "bay" instead. iirc existing mission texts referred to "cargo bay" a few times.

Reply:
Feel free to suggest new changes EjoThims I am a bit lost in these proper english arguments x)

Reply:
I like Todd's suggestion above.

Reply:
Should I just resolve this?

Reply:
```suggestion
	description "	As it is not meant to plunder but to lead in glorious battle, much of the cargo bays were removed and converted to space for weapons and defensive improvements. This change came at the cost of increased mass and lowered agility, but that is a sacrifice the Unfettered are more than willing to make."
```
like this then right?

Reply:
Looks good.